Revise FIPS compliance FAQ in fips-release.mdx: Update the fips relea… (#1212)
TykTechnologies/tyk-docs: developer-support/release-types/fips-release.mdx

## FAQ

*Tyk provides a FIPS-compliant package of the Tyk Gateway (Enterprise Edition) and Tyk Pump (together, the FIPS Tyk Product). Please note that the FIPS Tyk Product has not been submitted to a **[NIST](https://www.nist.gov/federal-information-processing-standards-fips)** testing lab for validation and Tyk is not FIPS certified..*\

> *FIPS-compliant means that the FIPS Tyk Product only uses FIPS 140-2 approved cryptographic algorithms (see below) when running in FIPS mode. This is only available to specific Tyk-built RPM and DEB packages FIPS Tyk Product. These packages and images are not publicly accessible.*

Reviewer: ```suggestion
*FIPS-compliant means that the Tyk components only use FIPS 140-2 approved cryptographic algorithms (see below) when running in FIPS mode. This is only available with the specific Tyk-built RPM and DEB packages that comprise the FIPS Tyk Product. These packages and images are not publicly accessible.*
```

-   *The FIPS Tyk Product uses the **[BoringCrypto module](https://boringssl.googlesource.com/boringssl/+/master/crypto/fipsmodule/FIPS.md#fips-140_2)**, enabling only FIPS 140-2 approved algorithms when run in FIPS mode.*

> -   *These packages are distributed separately, identifiable by the *`*-fips*`* suffix (e.g., *`*tyk-gateway-fips*`*, *`*tyk-pump-fips*`*).*

Reviewer: ```suggestion
-   *The FIPS Tyk Product packages are distributed separately and are identifiable by the *`*-fips*`* suffix (e.g., *`*tyk-gateway-fips*`*, *`*tyk-pump-fips*`*).*
```
